proposed/http-client/http-client-meta.md

```diff
@@ -39,10 +39,8 @@ specify the default behaviours.
 ### Default behavior
 
 The intention of this PSR is ensure library developers that all HTTP clients have the same 
-**default behavior**. That means that all HTTP clients MUST follow Liskov substitution principle
-when no configuration is provided. The PSR does not try to restrict nor define configuration for 
-HTTP clients. An implementing library is free to be configured by the application author to follow
-redirects, to throw exceptions or any other possible setting.  
+behavior. That means that all HTTP clients MUST follow Liskov substitution principle. 
```

Comment:
This is kind of obvious to me, I mean that's why we want to create this standard. Also, the next sentence kinda goes against this statement.

So I would rather say something about not restricting nor defining a configuration interface, but clients need to follow the contract by default (which might definitely be more than just fulfilling the interface, see exception handling, etc) even if they offer options that might break it.

Not sure though if it's a good idea to mention breaking the contract at all, so we could just leave this section out?

Reply:
i like the phrase below as it is now, solves the dilemma well for me.